docs/topology.md

```diff
@@ -43,3 +43,27 @@ than `kubenet`.  Currently the supported options are:
 - cni
 
 More information about [networking options](networking.md) can be found in our documentation.
+
+## Changing Topology of the API server
+To change the ELB that fronts the API server from Internet facing to Internal only there are a few steps to accomplish
+
+The AWS ELB does not support changing from internet facing to Internal.  However what we can do is have kops recreate the ELB for us.
+
+### Steps to change the ELB from Internet-Facing to Internal
+- edit the cluster `kops edit cluster $NAME`
+- change the api loadbalanver type from: Public to Internal... should look like this when done:
```

Comment:
@chrislovecnm - I don't know how kops replace would do this, I'm happy to give it a try however what I've documented works for sure.
@geojaz - will be fixed in my next commit.